File: server/groute/src/main/java/com/ssafy/groute/service/ThemeService.java
package com.ssafy.groute.service;

import com.ssafy.groute.dto.Theme;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public interface ThemeService {
    void insertTheme(Theme theme) throws Exception;
    Theme selectTheme(int id) throws Exception;
    List<Theme> selectAllTheme() throws Exception;
    void deleteTheme(int id) throws Exception;
    void updateTheme(Theme theme) throws Exception;
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/service/ThemeService.kt
package com.ssafy.groute.src.service

import com.ssafy.groute.src.dto.Theme
import com.ssafy.groute.util.RetrofitUtil
import retrofit2.Response

class ThemeService {
    suspend fun getThemeById(id: Int) : Response<Theme> {
        return RetrofitUtil.themeService.getThemeById(id)
    }
}